fuzzers/neuzz/fuzzer.py

```diff
@@ -91,8 +91,8 @@ def run_neuzz(input_corpus,
         'python2', './nn.py', '--output-folder', afl_output_dir, target_binary
     ]
     print('[run_neuzz] Running command: ' + ' '.join(command))
-    with subprocess.Popen(command, stdout=output_stream, stderr=output_stream):
-        pass
+    # pylint: disable=consider-using-with
```

Comment:
I think this patch changes too much. I think with usage didn't make sense in the first location, but I don't see why you would change these other ones. I think it makes sense here.

Reply:
Right, I think I should have considered this case by case:
1. Should use context manager (`with`) to wait for the process :
    * Here, to enable `asan` (probably can remove `sleep`, but I suppose it is better to leave it in this PR?).
2. Should not use context manager (`with`):
    * `gcloud`, to fix the bug in local experiments.
3. Neutral:
    * All others, using context manager has a neutral effect.

I am inclined to not use `with` in 3, because it looks cleaner, and it was that way before I [attempt to use `with` to fix linter errors](https://github.com/google/fuzzbench/pull/1529/files#r1003870199).
@jonathanmetzman what would you think? Thanks!

Reply:
I think you should get rid of it in gcloud.py.
I think the other cases should be unchanged. but I'm not sure about them, maybe try a make test-run-... to check them. Either way these files are unimportant so just fix gcloud for now and ignore these please.
Thanks!

Reply:
I see!
Done, thanks!